src/components/command-bar/commands/search/components/no-results-message/index.tsx

```diff
@@ -3,14 +3,56 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+import { ReactElement } from 'react'
+import { SearchableContentType } from 'contexts'
 import Text from 'components/text'
+import { useCommandBar } from 'components/command-bar'
 import s from './no-results-message.module.css'
 
-const NoResultsMessage = () => {
+interface NoResultsMessageProps {
+	tabsWithResults: {
+		type: SearchableContentType
+		heading: string
+		icon: ReactElement<React.JSX.IntrinsicElements['svg']>
+	}[]
+	currentTabHeading: string
+}
+
+/**
+ * Renders a "no results" message, that optionally directs visitors
+ * to other content-type tabs with results, if such results are present.
+ */
+function NoResultsMessage({
+	tabsWithResults,
+	currentTabHeading,
+}: NoResultsMessageProps) {
+	const { currentInputValue } = useCommandBar()
```

Comment:
We use `currentInputValue` from the command bar context to show the current query, so we can say "no results for X query" rather than just "no results". I think the intent here is to help users see things like typos, by making the query text a little more visible when there are no results.